packages/bigtable/src/table.js

```diff
@@ -861,8 +867,11 @@ Table.prototype.mutate = function(entries, callback) {
     })
   ]);
 
-  if (isStreamMode) {
-    return stream;
+  if (!isCallbackMode) {
+    ['error', 'finish'].forEach(function(event) {
```

Comment:
[This whole stream](https://github.com/callmehiphop/gcloud-node/blob/4125d3f647f43ed9e3749e70a27432a9efbaed4e/packages/bigtable/src/table.js#L846-L866) is where you can emit anything that needs to happen.

The stream linked above doesn't get piped to anything, so it doesn't need to push data onto itself. It should just flush the data, look for errors and emit them, then emit `complete` (not finish) when it's done.